countdigits.py

```diff
@@ -0,0 +1,7 @@
+num=input("Enter number : ")

```

Comment:
The input needs to be typecasted to number first, if it is successful then do the further steps, else raise an exception